### ISSUE TITLE
`Str.pascal()` method ignores uppercases  on the middle of words

### ISSUE THREAD

**Issue by rafapaulin** [supercharge/strings#63]
```
import Str from '@supercharge/strings';

Str('thisIsInCamelCase').pascal().get(); // returns 'Thisisincamelcase'. It should be 'ThisIsInCamelCase'

**Comment by rafapaulin** [comment]
similar issue with other naming convention methods. the only one that seems to work properly is `camel()`

**Comment by marcuspoehls** [comment]
@rafapaulin Hey Rafael, thanks for reporting this! I’ll have a look tomorrow. Maybe you found the issue already and want to submit a pull request.

**Comment by marcuspoehls** [comment]
Should be fixed in version `2.0.0`
